recipe/meta.yaml

```diff
@@ -20,14 +20,12 @@ requirements:
     - python                                # [build_platform != target_platform]
     - cython                                # [build_platform != target_platform]
     - meson-python                          # [build_platform != target_platform]
-    - ninja                                 # [build_platform != target_platform]
-    - scikit-learn >=1.5.0,<1.6             # [build_platform != target_platform]
```

Comment:
No scikit-learn code is executed at build time. Cython just uses the versions meant for the host to compile.

Thus only a `host` dependency.